src/examples/primitive_constraint_system.ts

```diff
@@ -0,0 +1,66 @@
+import { Field, Group, Poseidon, Provable, Scalar } from 'snarkyjs';
+
+function mock(obj: { [K: string]: (...args: any) => void }, name: string) {
```

Comment:
We didn't really have any regression tests that tested `Group` and its constraints, I just hacked this mock contract-like thing together to integrate it into the contract regression tests